public/literals.json

```diff
@@ -7,5 +7,6 @@
   "webgl_warning": "⚠️ There is a problem with your current configuration (WebGL is disabled or unavailable). The map won't be able to load.",
   "deprecated_layers_warning": "The activity layer $LAYERS used on this workspace is deprecated and may not get updated data, click <a href='http://globalfishingwatch.org/map'>here</a> to get the latest one.",
   "deprecated_layers_warning_plural": "The activity layers $LAYERS used on this workspace are deprecated and they may not get updated data, click <a href='http://globalfishingwatch.org/map'>here</a> to get the latest ones.",
-  "edge_warning": "Map performance might be degraded using Edge. We recommand using another browser such as Firefox or Chrome."
+  "edge_warning": "Map performance might be degraded using Edge. We recommand using another browser such as Firefox or Chrome.",
+  "vessel_warning": "This vessel has comments, please review them"
```

Comment:
@j8seangel I would rephrase to `Our analysts have reviewed this vessel and added some comments about it's behavior, please review them in the vessel info.`

Reply:
@j8seangel @enriquetuya My idea was to show the analyst's comment directly on the warning, so I thought this message was temporary. We can add it to the vessel info as well, I like that.